recipes/abyss/1.5.2/meta.yaml

```diff
@@ -33,9 +33,9 @@ requirements:
 
 test:
   commands:
-    - ABYSS-P --version
-    - ABYSS --version
-    - abyss-pe help
+    - ABYSS-P --version | grep "{{ version }}" > /dev/null
```

Comment:
Instead of `grep ... > /dev/null` you can also just do `grep -q ...`.

Reply:
Ahh, I was just pulling the style from other recipes, but `grep -q` does seem better.